builder/comp-builder.nix

```diff
@@ -167,7 +169,8 @@ let
 in stdenv.lib.fix (drv:
 
 stdenv.mkDerivation ({
-  name = "${ghc.targetPrefix}${fullName}";
+  name = nameOnly;
```

Comment:
I don't think this is right: the `name` argument should be `${name}-${version}`. With newer versions of nixpkgs you can pass the name part only as `pname`.